docs/reference/Collection.md

```diff
@@ -76,15 +76,15 @@ all embeddings up to limit starting at offset.
 - `where` - A Where type dict used to filter results by. E.g. `{"color" : "red", "price": 4.20}`. Optional.
 - `limit` - The number of documents to return. Optional.
 - `offset` - The offset to start returning results from. Useful for paging results with limit. Optional.
-- `where_document` - A WhereDocument type dict used to filter by the documents. E.g. `{$contains: {"text": "hello"}}`. Optional.
```

Comment:
this syntax was just wrong? its wrong in the underlying code https://github.com/chroma-core/chroma/blob/d16659b805a6078ba0177008582a017cc30a4f3f/chromadb/api/models/Collection.py#L121